### PR TITLE
fix: add full ID in inbox endpoint

### LINKED ISSUE #46: ID for Inbox is incomplete
It's missing "https://social.distributed.press" from it, only shows a path. Probably others endpoints too.

### PR DESCRIPTION
**Pull request by catdevnull**
fixes #46

### CODE REVIEW

File: src/server/api/inbox.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const inboxRoutes = (cfg: APIConfig, store: Store, apsystem: ActivityPubS
     const orderedCollection: APOrderedCollection = {
       '@context': 'https://www.w3.org/ns/activitystreams',
       type: 'OrderedCollection',
-      id: request.url,
+      id: `${cfg.publicURL}${request.url}`,
```
Comment (fauno) [inline src/server/api/inbox.ts:39]: we should document publicURL can't end with a slash, no?

Comment (RangerMauve) [reply src/server/api/inbox.ts:39]: good point, we can use the `Url` class here

Comment (RangerMauve) [inline src/server/api/inbox.ts:39]:
```suggestion
      id: new URL(request.url, cfg.publicURL).href,
```

**Review by RangerMauve — CHANGES_REQUESTED** [review]

**Review by RangerMauve — APPROVED** [review]
This means we can't support stuff like nginx configs with a subpath but that's okay

**Pull request merged**